### ISSUE TITLE
Orca 2.14.0 and newer: access denied on tasks requested from UI 

### ISSUE THREAD

**Issue by emptywee** [spinnaker/spinnaker#1834]
*Title*

After upgrading from 2.13.5 to 2.14.0+ any task requested from the UI gets access denied error.

*Cloud Provider*

Kubernetes

*Environment*

Self-hosted kubernetes cluster (Core OS) and spinnaker running on it.

*Feature Area*

UI, Orchestration

*Description*

Requesting to destroy a server group fails with a "access denied" error. Logs show:
```
[spin-orca-0] 2017-07-11 01:22:33.195  INFO 1 --- [pool-2-thread-1] c.n.spinnaker.orca.q.QueueProcessor      : [] Received message StartExecution(executionType=class com.netflix.spinnaker.orca.pipeline.model.Orchestration, executionId=c464aaec-cf64-4567-a6ed-9445264b92b8, application=proquote4)
[spin-orca-0] 2017-07-11 01:22:33.209  INFO 1 --- [pool-2-thread-1] c.n.spinnaker.orca.q.QueueProcessor      : [] Received message StartStage(executionType=class com.netflix.spinnaker.orca.pipeline.model.Orchestration, executionId=c464aaec-cf64-4567-a6ed-9445264b92b8, application=proquote4, stageId=788e3af1-51ef-42d0-ae5e-504d091e285c)
[spin-orca-0] 2017-07-11 01:22:33.226  INFO 1 --- [eHandlerPool-20] c.n.s.orca.clouddriver.OortService       : [] ---> HTTP GET http://spin-clouddriver:7002/applications/proquote4/clusters/rnq-cluster/proquote4-ws-c-dev4/kubernetes/serverGroups/proquote4-ws-c-dev4-v015
[spin-orca-0] 2017-07-11 01:22:33.245  INFO 1 --- [eHandlerPool-20] c.n.s.orca.clouddriver.OortService       : [] <--- HTTP 403 http://spin-clouddriver:7002/applications/proquote4/clusters/rnq-cluster/proquote4-ws-c-dev4/kubernetes/serverGroups/proquote4-ws-c-dev4-v015 (18ms)
[spin-orca-0] 2017-07-11 01:22:33.264 ERROR 1 --- [eHandlerPool-20] c.n.s.orca.q.handler.StartStageHandler   : [] Error running destroyServerGroup stage for Orchestration[c464aaec-cf64-4567-a6ed-9445264b92b8]
[spin-orca-0] retrofit.RetrofitError: 403 Forbidden
```

However, based on the logs orca sees who requested the task. When it passes it to Clouddriver it shows no user/anonymous as the requester.

*Steps to Reproduce*

Update orca to 2.14.0+ and request removing a server group.

*Additional Details*

See above.
Running components:
echo:v1.141.0
clouddriver:v1.650.0
gate:v3.46.0
fiat:v0.28.0
front50:v1.100.0
deck:v2.1126.0
igor:v1.71.0
orca: tried on 2.14.0 through 2.15.1

More information on request.

**Comment by emptywee** [comment]
Verified with v2.18.x - looks good. Please, hold off on this until I confirm 100%.

**Comment by emptywee** [comment]
Now it is not working again. Not sure how it worked last night. Any help in troubleshooting/debugging will be appreciated.

**Comment by emptywee** [comment]
orca v2.13.5 is working very well in this regards, providing stable positive results.

**Comment by emptywee** [comment]
Seems like there's a PR for exactly this: https://github.com/spinnaker/orca/pull/1439

**Comment by emptywee** [comment]
Closing as it's been fixed in orca v2.19.1
